src/commands/publish_images.yml

```diff
@@ -7,17 +7,22 @@ parameters:
     type: string
   organization:
     description: Registry organization
-    type: string
+    type: env_var_name
```

Comment:
Not sure why we need to change this to an environment variable. Environment variables are handy for injecting secrets (this isn't the case here) but it is harder to follow IMO as it can be overriden at various levels.

Reply:
The default registry organisation is currently `prom`. We need to push to `prombench` organisation. I will see if I can override it.

Reply:
My take is that rather than having an indirect reference through environment variable, it's more readable to pass the value from prombench's `.circleci/config.yml`:

```
...
    - prometheus/publish_images:
        login_variable: DOCKER_LOGIN
        password_variable: DOCKER_PASSWORD
        organization: prombench
        registry: docker.io
...
```

Reply:
I will change this.